unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/LoadingHUD/Bridge/LoadingBridge.cs

```diff
@@ -12,6 +12,8 @@ public class Payload
         public bool showTips = false;
     }
 
+    public event Action<bool> OnLoadingScreenVisibleStateChange;
```

Comment:
perhaps you can avoid creating a new event by subscribing to `DataStore.i.HUDs.loadingHUD.visible` changes.. anyway its up to you LGTM!

Reply:
Totally, I can't believe I didn't see that a couple of lines above my event call T_T, I'll change that, thanks!